src/patternfly/components/ExpandableSection/expandable-section.scss

```diff
@@ -112,6 +112,10 @@
       overflow: hidden;
     }
   }
+
+  &.pf-m-detached .pf-c-expandable-section__toggle {
```

Comment:
Since `--pf-c-expandable-section__toggle--MarginTop` is defined on `.pf-m-toggle`, you can redefine it on `.pf-m-detached` since it comes later in the CSS.

If you wanted to be super sure to only remove the top-margin from the truncate variation (assuming we add other variations that also use a this toggle--MarginTop var), you could set `--pf-c-expandable-section--m-truncate__toggle--MarginTop: 0;` here instead. That may be safer.